NRLMMD-GEOIPS/geoips
# v1.6.1 geoips - VERSION RELEASE - update formatting, test full install, bug fixes

# Related Issues
required to close NRLMMD-GEOIPS/geoips#62

# Reviewer Instructions
* Please confirm most recent completed pull request is labeled "DEVELOPMENTAL UPDATE" or "VERSION RELEASE":
    * https://github.com/NRLMMD-GEOIPS/geoips/pulls?q=is%3Apr+is%3Aclosed
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates.

# Summary
Changes for version 1.6.1 update on repo geoips.
* Test full install option
* Update formatting
* Add hooks for output formats without pre-determined products
* Bug fixes

See NRLMMD-GEOIPS/geoips#62 for additional repositories included in this update.

# Testing Instructions

Install GEOIPS and clone v1.6.1-release branch for:
* geoips


Tests should return 0:
export GEOIPS_ACTIVE_BRANCH=v1.6.1-release
$GEOIPS_PACKAGES_DIR/geoips/tests/test_full_install.sh

# Output
See NRLMMD-GEOIPS/geoips#62

# Post Merge Steps
Once this PR has been merged, v1.6.1 will be tagged/released on the geoips main branch.
